.github/workflows/ci.yml

```diff
@@ -312,15 +308,11 @@ jobs:
       - sequentialise
     steps:
       - name: Free up disk space
-        # see https://github.com/litentry/litentry-parachain/actions/runs/5654952764/job/15319056206
-        # before: 23 GB | after: 48 GB
         if: startsWith(runner.name, 'GitHub Actions')
-        run: |
-          sudo rm -rf /usr/local/lib/android
-          sudo rm -rf /usr/share/dotnet
-          sudo rm -rf /opt/ghc
-          sudo rm -rf /usr/local/share/boost
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: true
+          swap-storage: false
```

Comment:
Have you considered setting `large-packages` to `false` ?
We can exchange few GB for run minutes.

I've set it to false, here are the results:

Comparsion:
old: https://github.com/litentry/litentry-parachain/actions/runs/6296965077/job/17095165858
new: https://github.com/litentry/litentry-parachain/actions/runs/6296504239/job/17091914474


parachain-build-dev:
old: 9m 41s
new: 1m 44s

parachain-build-tee-prod:
old: 3m 7s
new: 16s

tee-build:
old: 2m 9s
new: 19s

**Total time saved: 12m 38s**

Reply:
woo. impressive.

Reply:
Thanks! I kept that because I fell short in disk after building the docker (once only 2-3 G left), but it seems to suffice in your case.

I'll change it in my next PR